Services/BrokerService/Utils/BrokerFactory.cs

```diff
@@ -1,13 +1,25 @@
-﻿using System;
+﻿using DTO;
 using Interfaces;
+using Kernel.CustomExceptions;
+using TinkoffIntegrationLib;
+using Microsoft.Extensions.Logging;
+using BrokerService.Controllers;
 
 namespace BrokerServices.Utils
 {
     public static class BrokerFactory
     {
-        public static IBroker Create(BankType bankType)
+        public static IBroker Create(BankType bankType, ILogger<BrokerController> logger, BrokerData brokerData)
         {
-            throw new NotImplementedException();
+            switch (bankType)
+            {
+                case BankType.TinkoffBank:
+                    logger.LogInformation("Broker successfully created.");
```

Comment:
I think, it`s not a good log. You write, that Broker was created before creating broker